.pre-commit-config.yaml

```
@@ -5,28 +5,12 @@ default_language_version:
-      - id: black
-  - repo: https://github.com/asottile/pyupgrade
-    rev: v3.19.0
-    hooks:
-      - id: pyupgrade
-        args: [--py39-plus]
+- repo: https://github.com/astral-sh/ruff-pre-commit
+  # Ruff version.
+  rev: v0.8.0
+  hooks:
+    # Run the linter.
+    - id: ruff
```

yamllint (1.29.0-1)

[warning] wrong indentation: expected 2 but found 4

(indentation)